scipy/spatial/transform/_rigid_transformation.pyx

```diff
@@ -927,6 +976,8 @@ cdef class RigidTransformation:
             real_part = np.roll(real_part, -1, axis=1)
             dual_part = np.roll(dual_part, -1, axis=1)
 
+        real_part, dual_part = _normalize_dual_quaternion(real_part, dual_part)
```

Comment:
At the moment we normalize by default. Should we make this configurable like in the constructor?

Reply:
I think keep the normalization by default, `Rotation.from_quat()` automatically normalizes